docs/70-early_stakers.md

```diff
@@ -4,12 +4,12 @@ description: Early Stakers can now deposit ETH or stETH prior to Diva's launch t
 ---
 # Early Stakers Initiative
 
-Diva is still under development, but you can already support its launch.
+Diva Staking is still under development, but you can already support its launch.
 
 **You can now deposit ETH or stETH into [Pre-Launch Vaults](https://diva.enzyme.finance/#vaults) to become eligible for DIVA tokens**.
 
 :::info Good to know
-This is currently the only way to earn DIVA tokens, as they're otherwise not transferable
+DIVA tokens are **currently** non-transferable. Join the initiative and community for potential DIVA allocations.
```

Comment:
"Join this iniative" should be enough (its not technically needed to join the community)

Reply:
i propose to link [currently non-transferable](https://www.tally.xyz/gov/diva/proposal/52481024395238134144299582623582875841236980209822828761178984408970724801644) so people can click and learn

Reply:
Let's link 'non-transferable' only as currently is formatter in **bold**?

Reply:
ok!

Reply:
Updated